package io.github.vxrpenter.secretlab.exceptions

class CallFailureException(message: String, cause: Throwable? = null) : SecretLabException(message, cause)